sippy-ng/src/component_readiness/CompReadyMainInputs.js

```diff
@@ -33,6 +35,33 @@ const useStyles = makeStyles((theme) => ({
 export default function CompReadyMainInputs(props) {
   const theme = useTheme()
   const classes = useStyles(theme)
+  // checkBoxGroupByHiddenVariants defines what variants are excluded as GroupBy variants
+  const checkBoxGroupByHiddenVariants = new Set([
+    'FromRelease',
+    'FromReleaseMajor',
+    'FromReleaseMinor',
+    'Release',
+    'ReleaseMajor',
+    'ReleaseMinor',
+  ])
+  // checkBoxHiddenIncludeVariants defines what variants are excluded when creating Include Variant CheckBox
+  const checkBoxHiddenIncludeVariants = new Set([
+    'Aggregation',
+    'FeatureSet',
+    'FromRelease',
+    'FromReleaseMajor',
+    'FromReleaseMinor',
+    'NetworkAccess',
+    'NetworkStack',
+    'Owner',
+    'Release',
+    'ReleaseMajor',
+    'ReleaseMinor',
+    'Scheduler',
+    'SecurityMode',
+    'Suite',
+    'Topology',
+  ])
```

Comment:
Not required here unless you think it's easy doable, but I think this should be driven by the server as much as possible.  Keeping productization in mind, someone else may have an entirely different set of variants